src/prog_models/models/aircraft_model/vehicles/vehicles.py

```diff
@@ -174,16 +176,13 @@ def rotorcraft_inertia(m, g):
     # Define rotor positions on the 360 degree circle
     # ------------------------------------------------
     angular_vector = rotor_angles(n_rotors)
-    
+
     motor_distance_from_xaxis = g['arm_length'] * np.sin(angular_vector)
-    if g['body_type'].lower() == 'sphere':
-        Ix0, Iz0 = sphere_inertia(m['body'], g['body_radius'])
-    elif g['body_type'].lower() == 'flatdisk':
-        Ix0, Iz0 = flatdisk_inertia(m['body'], g['body_radius'])
-    elif g['body_type'].lower() == 'thickdisk':
-        Ix0, Iz0 = thickdisk_inertia(m['body'], g['body_radius'], g['body_height'])
-    else:
-        raise Exception("Body geometry not implemented. Please choose among: sphere, flatdisk, thickdisk.")
+    if g['body_type'].lower() == 'sphere': Ix0, Iz0 = sphere_inertia(m['body'], g['body_radius'])
+    elif g['body_type'].lower() == 'flatdisk': Ix0, Iz0 = flatdisk_inertia(m['body'], g['body_radius'])
+    elif g['body_type'].lower() == 'thickdisk': Ix0, Iz0 = thickdisk_inertia(m['body'], g['body_radius'], g['body_height'])
+    else: raise Exception("Body geometry not implemented. Please choose among: sphere, flatdisk, thickdisk.")
```

Comment:
```suggestion
    if g['body_type'].lower() == 'sphere':
        Ix0, Iz0 = sphere_inertia(m['body'], g['body_radius'])
    elif g['body_type'].lower() == 'flatdisk':
        Ix0, Iz0 = flatdisk_inertia(m['body'], g['body_radius'])
    elif g['body_type'].lower() == 'thickdisk':
        Ix0, Iz0 = thickdisk_inertia(m['body'], g['body_radius'], g['body_height'])
    else:
        raise Exception("Body geometry not implemented. Please choose among: sphere, flatdisk, thickdisk.")
```
I find this easier to read